package Listener;

import org.testng.IRetryAnalyzer;
import org.testng.ITestResult;

public class RetryAnalzer implements IRetryAnalyzer {

    private static final int MAX_RETRIES = 1;
    private int attempts = 0;

    @Override
    public boolean retry(ITestResult result) {
        if (attempts < MAX_RETRIES) {
            attempts++;
            return true;
        }
        return false;
    }
}
